sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -53,8 +55,12 @@ public TimeSpan MaxAutoRenewDuration
             }
         }
 
-        /// <summary>Gets or sets the time to wait for receiving a message.</summary>
-        /// <value>The time to wait for receiving the message.</value>
+        /// <summary>Gets or sets the timeout to wait for receiving a message. If a session is running out of active messages, it will wait up 
+        /// to this value to close the current session. Once the session is closed, the session lock will be lost and can no longer receive new messages. 
+        /// This value has an impact on the message throughput. If the value is very large, then every time when the traffic in a session is slowing down, 
+        /// it needs to wait for this amount of time before closing to make sure that no more messages will arrive. If users are using session in a 
+        /// high-throughput scenario, try setting this to be a relative smaller value based on how frequent new messages arrive in the session. </summary>
```

Comment:
There are two separate scenarios - 
1. Few sessions and high number of messages per session.
2. Loads of sessions and fewer messages per session.

Mainly in the latter scenario this will affect throughput drastically. The time is essentially the "time to switch between sessions". If we mention "high-throughput scenario", it might be mistaken even for the first scenario. Instead we need to make it clear and mention that it waits for this duration before switching to a different session, and hence can observe low throughput in cases where there are a lot of sessions.